site/src/pages/_app.page.tsx

```diff
@@ -30,6 +31,12 @@ const MyApp = ({
 }: MyAppProps) => {
   const router = useRouter();
 
+  useEffect(() => {
+    if (process.env.NEXT_PUBLIC_VERCEL_ENV === "production") {
```

Comment:
Just to confirm my understanding, `process.env.NEXT_PUBLIC_VERCEL_ENV` is not equivalent to `process.env.NODE_ENV` as `NODE_ENV === "production"` for all deployed branches, but `NEXT_PUBLIC_VERCEL_ENV === "production"` only on the `main` branch deployment?

Reply:
Correct - it's one of `production`, `preview`, or `development`, with `production` being the main deployment in Vercel.